scripts/billing/sql/total_costs_by_label_value.sql

```diff
@@ -0,0 +1,13 @@
+/*
+Query returns total costs by label values for a specific key
+
+This example uses a label key of "environment" as an example
+*/
+
+SELECT
```

Comment:
This query will return costs for labels.key is NULL, is it intended.

If not, please put the filter labels.key to where clause.

Reply:
Nice catch. Updated to filter NULL keys

Reply:
Added filter for null key values